core-bundle/tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -252,9 +252,6 @@ public function testValidCronConfiguration(array $params, bool|string $expected)
         $this->assertSame($expected, $configuration['cron']['web_listener']);
     }
 
-    /**
-     * @dataProvider cronConfigurationProvider
-     */
```

Comment:
This was wrong anyway. It runs the same test 3 times for nothing. This provider belongs to the test above. Copy & paste issue.